dips/dip-20.md

```diff
@@ -319,32 +297,11 @@ privilege](https://github.com/diem/dip/blob/master/dips/dip-2.md#permissions)
 for the currency of the coins being moved--granted to the currency's
 reserve management--removing them once a redemption has been completed.
 
-The specific set of operations that may be performed that relate to
-minting, preburning, and burning of Diem Coins are as follows.
-
 ### <a name="minting">Minting</a>
 
-Diem Coins in a currency `C` with a non-zero value may be created by calling
-
-```rust
-Diem::mint_with_capability<C>(value: u64, capability: MintCapability<C>): Diem<C>
-```
-
-A reference to the `MintCapability<C>` resource for the currency being minted
-must be passed-in to this function as a witness of having the privilege to mint
-coins in the specific currency.
-
-Additionally, non-zero-value coins for SCS currencies can be minted by calling
-
-```rust
-Diem::mint<C>(value: u64): Diem<C>
-```
-
-The `mint` function may only be called by an account with the
-[Treasury Compliance role](https://github.com/diem/dip/blob/master/dips/dip-2.md#roles)
-that has a `MintCapability<C>` resource published under it. Recall that only
-one mint capability resource is created for a currency when it is
-registered.
+Any account that has a [`MintCurrency<C>`](https://github.com/diem/dip/blob/master/dips/dip-2.md#permissions)
+privilege may mint, or create, a coin with a non-zero value.
+Recall that there is only one minting permission for each currency that can exist.
```

Comment:
Dangling phrase? Might be better as "Recall that only one minting permission can exist for each currency."?